app/views/planning_applications/review_form.html.erb

```diff
@@ -18,27 +20,39 @@
   </p>
 
   <%= render "recommendations", { recommendations: @planning_application.recommendations } %>
-  <%= form_for @recommendation, url: review_planning_application_path(@planning_application) do |form| %>
-    <div class="govuk-form-group <%= form.object.errors.any? ? 'govuk-form-group--error' : '' %>">
-      <p class="govuk-body">Do you agree with the recommendation?</p>
 
-      <div class="govuk-radios govuk-radios--conditional" data-module="govuk-radios" style="margin-bottom: 20px;">
-        <div class="govuk-radios govuk-radios--small">
-          <div class="govuk-radios__item">
-            <%= form.radio_button :agree, "Yes", class: "govuk-radios__input" %>
-            <%= form.label :agree_yes, "Yes", class: "govuk-label govuk-radios__label" %>
-          </div>
-          <div class="govuk-radios__item">
-            <%= form.radio_button :agree, "No", class: "govuk-radios__input" %>
-            <%= form.label :agree_no, "No", class: "govuk-label govuk-radios__label" %>
+  <%= form_for @recommendation, url: review_planning_application_path(@planning_application) do |form| %>
```

Comment:
I don't know if we want to keep this consistent, but we are using GOVUKDesignSystemFormBuilder::FormBuilder for some of our other forms?